functests/4_latency/latency.go

```diff
@@ -63,25 +63,19 @@ func init() {
 			klog.Fatalf("the environment variable LATENCY_TEST_DELAY has incorrect value %q", latencyTestDelayEnv)
 		}
 	}
-
-	maximumLatencyEnv := os.Getenv("OSLAT_MAXIMUM_LATENCY")
-	if maximumLatencyEnv != "" {
-		var err error
-		if maximumLatency, err = strconv.Atoi(maximumLatencyEnv); err != nil {
-			klog.Fatalf("the environment variable OSLAT_MAXIMUM_LATENCY has incorrect value %q", maximumLatencyEnv)
-		}
-	}
 }
 
 var _ = Describe("[performance] Latency Test", func() {
 	var workerRTNode *corev1.Node
 	var profile *performancev2.PerformanceProfile
-	var oslatPod *corev1.Pod
+	var latencyTestPod *corev1.Pod
+	var testName string
```

Comment:
Can `testName` be moved on each specific `Context`? IOW, does it need to be a top-level variable?